cypress/page/address-step.page.ts

```diff
@@ -0,0 +1,37 @@
+class AddressStepPage {
+    private addressButton: string;
+    private accountPageURL: string;
+
+    constructor() {
+        this.addressButton = ".cart_navigation > .button > span";
+        this.accountPageURL = "http://automationpractice.com/index.php?controller=order";
+    }
+
+    public visitloginPageURL(): void {
+        cy.visit(this.accountPageURL)
+    }
```

Comment:
Are you using this method? I think it is not correct to have a cy.visit in middle of a test